docs/DeveloperGuide.adoc

```diff
@@ -361,22 +367,35 @@ Windows, Linux, Unix, OS-X
 [[private-contact-detail]] Private contact detail::
 A contact detail that is not meant to be shared with others
 
+[[logging]] Logging::
+Logging is the use of logging tools to track the performance of the application while it is running.
+
+[[participant]] Participant::
+It represents a participant taking part in the hackathon
+
+[[mentor]] Mentor::
+It represents a mentor available for teams to choose
+
+[[team]] Team::
+Team is the base unit of this project. It contains references to an associated list of participants and an optional mentor. 
+
 [appendix]
 == Product Survey
 
-*Product Name*
+*Google Sheets*
 
-Author: ...
+Author: Google
 
 Pros:
 
-* ...
-* ...
+* This is extremely versatile as Google Sheets come with a list of extremely helpful macros that could help in the storage of participants.
+* The display and UI of Google Sheets is extremely intuitive and will come as second nature to anyone using the web.
 
 Cons:
 
-* ...
-* ...
+* Google Sheets has no concept of objects and thus it cannot accurately depict the relationships between our different entities
+* Google Sheets may be useful for storing information, but it does not support command line arguments
+* Google Sheets is also unable to perform input validation as it lacks the logic to do so.
```

Comment:
Hard to locate/look for participant-team relationship or team-mentor relationship.